fuzz/Cargo.toml

```diff
@@ -1,8 +1,11 @@
 [package]
-name = "quinn-fuzz"
-version = "0.0.0"
-authors = ["Automatically generated"]
+name = "fuzz"
+version = "0.1.0"
+authors = [
+    "Benjamin Saunders <ben.e.saunders@gmail.com>", "Dirkjan Ochtman <dirkjan@ochtman.nl>",
```

Comment:
Feel free to add yourself here!

Reply:
:-D thank you I appreciate it but I am satisfied just being in the git log for now, if I put my name here then someone in the future may ask me to fix bugs!

Reply:
Haha, always something to be mindful of :D